### PR TITLE
Conditions are not classes and we can signal them!

### PR DESCRIPTION
**Pull request by jsjolen**
See https://docs.google.com/document/pub?id=1SYSr0KE8iyv61srv6AMm-YRr-O6zM7Oi6CLSJgZc9yc&pli=1

### CODE REVIEW

File: error_handling.md [hunk 1]
```diff
@@ -318,15 +317,13 @@ unknown (not previously defined with `define-opts') option."))
...
 ## Signaling (throwing) conditions
 
-We can use `error`, like we did above. Two ways:
+We can use `error`, like we did above, in two ways:
 
 - `(error "some text")`: signals a condition of type `simple-error`
 - `(error 'my-error :message "We did this and this and it didn't work.")`
 
 Throwing these conditions will enter the interactive debugger,
-where a few options will be presented by default. We can give
```
Comment (pnathan) [inline error_handling.md]: Maybe "use `signal` and allow the user to select a restart?"

Comment (jsjolen) [reply error_handling.md]: Agreed

**Comment by pnathan** [comment]
I haven't used Github a lot in the past year. Sorry about that, @jsjolen . mea culpa, mea maxima culpa...

**Comment by vindarel** [comment]
thanks for merging !

**Pull request merged**